tiling.js

```diff
@@ -3989,6 +3989,8 @@ export function insertWindow(metaWindow, options = {}) {
     };
 
     let overwriteSpace = undefined;
```

Comment:
Don't need `undefined` here (it's undefined by default) it's being flagged by the our linter.